Failed to download kubeconfig file when --kube-config-path destination exists

If I try to download kubeconfig file using a preexisting folder, command fails, but if folder is created by the command, it works

```
[morenod@morenod-laptop osd]$ mkdir /tmp/kubekube

[morenod@morenod-laptop osd]$ osde2e/out/osde2ectl --custom-config custom_config.yaml get -k -i 1hihgcluo1cnd0hqke7rvhla1jgcqm13 --kube-config-path /tmp/kubekube
2020/12/14 14:33:14 Custom YAML config provided, loading from custom_config.yaml
             NAME: osde2e-glgxq
               ID: 1hihgcluo1cnd0hqke7rvhla1jgcqm13
            STATE: installing
           STATUS: waiting-for-ready
            OWNER: dsanzmor
INSTALLED VERSION: openshift-v4.5.13
  UPGRADE VERSION: 
           JOB ID: -1
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x1a9a5e9]

goroutine 1 [running]:
github.com/openshift/osde2e/cmd/osde2ectl/get.run(0x30414a0, 0xc00044c4d0, 0x0, 0x7, 0x0, 0x0)
	/home/morenod/osd/osde2e/cmd/osde2ectl/get/cmd.go:166 +0x1889
github.com/spf13/cobra.(*Command).execute(0x30414a0, 0xc00044c380, 0x7, 0x7, 0x30414a0, 0xc00044c380)
	/home/morenod/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0x30419e0, 0x1, 0xc000337f78, 0x408005)
	/home/morenod/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/morenod/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
main.main()
	/home/morenod/osd/osde2e/cmd/osde2ectl/main.go:36 +0x31


[morenod@morenod-laptop osd]$ echo $?
2


[morenod@morenod-laptop osd]$ osde2e/out/osde2ectl --custom-config custom_config.yaml get -k -i 1hihgcluo1cnd0hqke7rvhla1jgcqm13 --kube-config-path /tmp/kubekube2
2020/12/14 14:33:48 Custom YAML config provided, loading from custom_config.yaml
             NAME: osde2e-glgxq
               ID: 1hihgcluo1cnd0hqke7rvhla1jgcqm13
            STATE: installing
           STATUS: waiting-for-ready
            OWNER: dsanzmor
INSTALLED VERSION: openshift-v4.5.13
  UPGRADE VERSION: 
           JOB ID: -1
File directory is invalid -  stat /tmp/kubekube2: no such file or directory Will create a new directory.
Successfully downloaded the kubeconfig into - /tmp/kubekube2/osde2e-glgxq-kubeconfig.txt . Run the command "export TEST_KUBECONFIG= /tmp/kubekube2/osde2e-glgxq-kubeconfig.txt "


[morenod@morenod-laptop osd]$ ls /tmp/kubekube/
[morenod@morenod-laptop osd]$ ls /tmp/kubekube2
osde2e-glgxq-kubeconfig.txt
```

--- a/cmd/osde2ectl/get/cmd.go
+++ b/cmd/osde2ectl/get/cmd.go
@@ -162,7 +162,8 @@ func run(cmd *cobra.Command, argv []string) error {
 				if err != nil {
 					return fmt.Errorf("Unable to create a new directory - %s", err.Error())
 				}
-			} else {
+			}
+			if err != nil {
 				return fmt.Errorf("Unable to stat file path: %s", err.Error())
 			}
 			filePath = filepath.Join(args.kubeConfigPath, filename)

